setup.py

```diff
@@ -2,6 +2,13 @@
 from codecs import open  # To use a consistent encoding
 from os import path
 
+
+def requirements_from_file(filename):
+    """Parses a pip requirements file into a list."""
+    return [line.strip() for line in open(filename, 'r')
+            if line.strip() and not line.strip().startswith('--')]
```

Comment:
For completeness, you may want to add comments management (strip `#.*`), but we don't really need it for now.

Reply:
Agreed. For future reference, if we want to parse a fully generic requirements file, the syntax is outlined [here](https://pip.pypa.io/en/latest/reference/pip_install/#requirements-file-format).